src/Die.java

```diff
@@ -0,0 +1,11 @@
+import java.util.Random;
+
+public class Die {
+
+    protected static int roll(){
+        Random rng = new Random();
```

Comment:
Het is beter om een random generator één keer te createn, dus je kan hier beter een (static) field van maken. Of [`Math.random()`](https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#random--) gebruiken.

Reply:
Thanks! I dacht dat ik dat al geprobeerd had, maar nu jij en Rick het zeiden leek het mij goed het nog eens in te typen. En ja hoor, het werkt idd op die manier. Het was mijn eerste idee om die niet vaker te createn en toe doen zoals jullie zeiden, echter het werkte toen niet.
Nah, nu wel 😄 

Thanks!

Reply:
> Of [`Math.random()`](https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#random--) gebruiken.

Is er een voordeel aan `Math.random`? Het lijk mij namelijk onoverzichterlijker dan `Random()` omdat ik die een start en een bound mee kan geven. in `Math.random `zou dat zo moeten `return (int) 1 + (Math.random() * 7`

Reply:
Ja, klopt helemaal! `Math.random()` is met name handig omdat je geen variabele/field hoeft te managen.